### ISSUE TITLE
[Help] Crashing with Desmume

### ISSUE THREAD

**Issue by TuckingFypoTsu** [libretro/RetroArch#3530]
So first things first I am on an Iphone 5s and Jailbroken (so I am always using the most up to date version of retroarch) IOS 9. The problem I'm dealing with is whenever I download and try to load either a zip or a 7z the app just crashes or long freezes. The only clue I've seen to fix this issue is to enable interpreter mode, however I don't see any of the options most people mention in the core menu (such as switching to interpreter mode). 

Lastly if someone already knows this fact then it would save me many more hours of trying to find a solution. Anybody know if 3D sections (in particular Phoenix Wright: Apollo Justice) work in the RetroArch Desmume? It doesn't work on iNDS or nds4ios, so Im just trying retroarch as a last ditch effort.

**Comment by twinaphex** [comment]
The current Desmume is unlikely to be fast enough on an iPhone 5s though.

**Comment by TuckingFypoTsu** [comment]
I don't know if this is relavent to what you said but i can run iNDS and nds4ios at 61fps no problem. And acording to the dev of iNDS he said the version of desmume that has the 3D sections fixed is 1 version above what iNDS uses currently.

**Comment by bparker06** [comment]
The current version of desmume in the libretro-desmume port is 0.9.11 so in theory it should work, although when I tried loading that game on a PC I just got a white screen and it would never load.

**Comment by TuckingFypoTsu** [comment]
Yup, that's the exact version that can run the game's 3D sections (or more of I should say recognize since it can "run" them). The dev cobb of iNDS said the 2 options to fix the issue is to go from 0.9.10 to 0.911 which apparently is hard for them right now or is to allow JIT which is the only difference between 32x devices and 64x devices right now. Atleast that's what I am assuming right now because my IPhone4 had no issue with playing the section because right now iNDS can only do interpreter mode on 64x devices. So right now it's frustrating. Play on my iPhone 5S and get 61fps but can't do 3D sections or play on iPhone 4S and deal with 30-40fps and terrible audio quality due to low fps. So luckily I know the game will work on RetroArch I just need help getting it to work. Instead of making a new forum however if possible lets just post the "help" on this one. As for actually running the game on PC I ran it fine 100% on desmume. Just beat the first chapter which has 2 3D sections and got by both of them easily.

Sidenote: I tried a work around with DropBox by trying to sync my iNDS's files to desmume's but it was just a clown fiesta of constant overriding in the "wrong/unintended" way. Closest thing to progress I could get was to get the game to load the specific save file from desmume but the game itself (not the emulator) couldn't recognize the save file so I gave up on that method.
